### PR TITLE
chore: release 0.33.0

### PR DESCRIPTION
**Pull request by thbcmlowk**
# Issue

Closes FL-1227

# Build

[docs](https://owkin-substra-documentation--370.com.readthedocs.build/en/370/additional/release.html)

### CODE REVIEW

File: docs/source/additional/release.rst [hunk 1]
```diff
@@ -31,6 +31,21 @@ This is an overview of the main changes, please have a look at the changelog of
...
+
+- **BREAKING**: Substra tools now only provides a unique ``latest`` docker image for each supported Python version.
+
+**Orchestrator and Backend:**
+
+- A new Skaffold profile ``three-orgs`` have been added in the Orchestrator and the Substra-backend. It allows local deployment of a third organization and must be activated on the two repositories. To use it:
+
+  .. code-block:: bash
+
+    skaffold run -p three-orgs
```
Comment (oleobal) [inline docs/source/additional/release.rst:47]: Is this relevant to users?

File: docs/source/additional/release.rst [hunk 2]
```diff
@@ -31,6 +31,21 @@ This is an overview of the main changes, please have a look at the changelog of
 - `backend changelog <https://github.com/Substra/substra-backend/blob/main/CHANGELOG.md>`__
 - `orchestrator changelog <https://github.com/Substra/orchestrator/blob/main/CHANGELOG.md>`__
 
+Substra 0.33.0 --- 2023-10-06
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Substra:**
+
+- **BREAKING**: Substra tools now only provides a unique ``latest`` docker image for each supported Python version.
```
[the remark this answers is not in the document]
Comment (thbcmlowk) [reply docs/source/additional/release.rst:39]: Fair point

[the remark this answers is not in the document]
Comment (thbcmlowk) [reply docs/source/additional/release.rst]: Updated, WDYT?

[the remark this answers is not in the document]
Comment (oleobal) [reply docs/source/additional/release.rst]: Fine by me!

File: docs/source/additional/release.rst (hunk 1 above)
Comment (oleobal) [reply docs/source/additional/release.rst:47]: I thought it was only meant for the doc examples in the CI.

I'm not sure this message deserves to be here but IMO if it does it should also be mentioned in [the guide](https://github.com/Substra/substra-documentation/blob/release-0.33.0/docs/source/how-to/developing-substra/local-deployment.rst) and this paragraph should make it clear it's not a new backend/orc feature but a new local dev feature.

Comment (thbcmlowk) [reply docs/source/additional/release.rst]: The CI was the need, but now that it's here I don't see why not using it. Second point is fair. Updated. WDTY?

**Review by ThibaultFy — APPROVED** [review]
In the examples of substra core, you need to change the substratools version in all dockerfiles

**Review by ThibaultFy — CHANGES_REQUESTED** [review]
In the examples of substra core, you need to change the substratools version in all dockerfiles

**Comment by thbcmlowk** [comment]
> In the examples of substra core, you need to change the substratools version in all dockerfiles
> 
> cmd+maj+f : ghcr.io/substra/substra-tools:0.20.0-

I don't find anything, can you provide an example?

File: docs/source/additional/release.rst (hunk 1 above)
Comment (oleobal) [reply docs/source/additional/release.rst]: 👍 ❗

**Comment by thbcmlowk** [comment]
> In the examples of substra core, you need to change the substratools version in all dockerfiles
> 
> cmd+maj+f : ghcr.io/substra/substra-tools:0.20.0-

😲 thank you

**Review by ThibaultFy — APPROVED** [review]
